ruma-events/src/room/relationships.rs

```diff
@@ -38,9 +38,11 @@ pub(crate) enum RelationJsonRepr {
     Annotation(Annotation),
     /// A reference to another event.
     #[serde(rename = "m.reference")]
+    #[cfg(feature = "unstable-pre-spec")]
```

Comment:
Sorry for the nitpick, but could you move the `cfg` attribute about the `serde` attribute? We generally have builtin / std attributes above custom ones.